__tests__/app.test.js

```diff
@@ -422,29 +422,3 @@ describe("/api/reviews/:review_id/comments", () => {
     });
   });
 });
```

Comment:
There's no test for the delete end point

Reply:
Seems like my tests got lost during the merge. I will rewrite them and submit the PR again when finished.